test/test_helper.rb

```diff
@@ -5,7 +5,7 @@
 require "active_record"
 require "active_job"
 
-ActiveJob::Base.logger.level = :warn
+ActiveJob::Base.logger.level = 1
```

Comment:
What’s the reason for this?

Reply:
I changed this because I got an `ArgumentError: comparison of Fixnum with :warn failed` while running the test suite. Which lead me to believe that the log level should be an integer.

Though it may have just been because I have a different version of ActiveJob or ruby (probably older). I can change it back if needed.